lnst/Common/Utils.py

```diff
@@ -95,9 +95,9 @@ def int_it(val):
 
 def bool_it(val):
     if isinstance(val, str):
-        if re.match("^\s*(?i)(true)", val) or re.match("^\s*(?i)(yes)", val):
+        if re.match("^\s*(true|yes)", val, flags=re.IGNORECASE):
```

Comment:
Perhaps we should also complile these regexes

Reply:
i wonder how much gain there would be from that... the function isn't really called that frequently that this is actually slowing us down...

i mean, we could as well compile all of the regexes that we use everywhere, but honestly as noted in https://docs.python.org/3/library/re.html?highlight=re%20compile#re.compile the recently used regexes are cached and don't really need to be compiled in most cases.

so i think keeping this simpler is fine since we're not really seeing any significant performance hit from this...

Reply:
Didnt realized python caches recent regexes, interesting. 

I agree its okay to keep it simple.